docs/docs-content/clusters/edge/trusted-boot/deployment-day2/install.md

```diff
@@ -0,0 +1,179 @@
+- A USB disk containing the installer ISO.
+
+:::warning
+
+Before proceeding with installation, check the maximum EFI size that your Edge host is able to boot and compare it
+against the installer ISO and provider image. With secure boot enabled and keys enrolled, you won't be able to boot from
+the EFI size checking ISO unless you disable secure boot or clear the keys. For more information, refer to
+[Check EFI Size and Edge Host Boot Limit](../edgeforge/check-efi-limit.md).
+
+:::
+
+## Instructions
+
+### Enroll Secure Boot Keys into Edge Device
+
+The following steps are written for the Intel NUC 13 Pro device. If you are using a different device, the exact sequence
+of steps might be different. However, you should be able to find the same or similarly worded options in any device that
+supports Secure Boot.
+
+1. Insert the USB disk into the Edge device.
+
+2. Power up your Edge device and enter the BIOS interface. Oftentimes you can accomplish this by pressing F2, F1, or F10
+   on the keyboard immediately after powering up the device and before you select a boot volume, as is the case in Intel
+   NUC 13 Pro. However, the exact method might be different on your Edge device. Consult the manufacturer of your Edge
+   device to find out how to enter the BIOS interface.
+
+3. From the BIOS menu, select the **Boot** tab. Then select **Secure Boot**.
+
+4. From the **Secure Boot Menu**, make sure that secure boot is enabled. Then select **Restore to Setup Mode**. Two
+   dialogue boxes will pop up to confirm this operation. Select **Yes** to confirm. This will make the device restart.
+
+   :::info
+
+   **User Mode** and **Setup Mode** are UEFI terms. Setting secure boot mode to **Setup Mode** means clearing the keys
+   that are currently in the device. This allows you to enroll your own keys, while **User Mode** means you have
+   enrolled your own keys.
+
+   On some Edge hosts, the same actions or functionality might be named differently in the BIOS interface. Consult the
+   manufacturer of your Edge host to find out how to clear the keys and enroll your own keys.
+
+   :::
+
+   If you set the `AUTO_ENROLL_SECUREBOOT_KEYS` argument to `true` when you built the installation ISO, key enrollment
+   will begin automatically the first time the device boots from the installer ISO. You can skip the following steps to
+   proceed to [Install Palette Edge with Trusted Boot](#install-palette-edge-with-trusted-boot). In Virtual Machines
+   (VM), keys will always be auto-enrolled.
+
+5. If you did not set `AUTO_ENROLL_SECUREBOOT_KEYS` to `true` when you built the ISO, you will need to select the
+   **Enroll Secure Boot keys: Auto** option in the GRUB menu when the Edge host restarts. This will start the key
+   enrollment process.
+
+   :::warning
+
+   There is currently a known issue where if `AUTO_ENROLL_SECUREBOOT_KEYS` is set to `false`, it's possible for manual
```

Comment:
We need to capture this in the global known issues page and add it to the release notes.